### PR TITLE
[CSMD-27] Develop an E-commerce Store Database

### PR DESCRIPTION
**Pull request by HalimahOladimeji**
I created a Database for my E-commerce Store.

### CODE REVIEW

File: submissions/HalimahOladimeji/Database/init.sql [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        
+	--// Inserted records into the categories table
+		INSERT INTO categories (category_name)
+			VALUES 	('samsung'), ('tecno'), ('apple'), ('infinix'),('redmi');
+                    
+--// PRODUCTS
+	--// Create products table
+		CREATE TABLE products (
+			product_id INT NOT NULL AUTO_INCREMENT,
+            category_id INT NOT NULL,
```
Comment (adaugochi) [inline submissions/HalimahOladimeji/Database/init.sql:39]: This line should be aligned with others

File: submissions/HalimahOladimeji/Database/init.sql [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+			VALUES 	('Joseph', 'Akinola', 'josephakin@gmail.com', 'bookworm24'),
+				('Ajayi', 'Coker', 'cokerajayi@gmail.com', 'snickers77'),
+				('Adeola', 'Adeoye', 'adeolaoye@yahoo.com', 'powderjuice888'),
+				('Matthew', 'Gbenga', 'gbengamatt@yahoo.com', 'bodycare20');
+                    
+--// CUSTOMERS CONTACTS 
+	--// Create the customers contacts table
+		CREATE TABLE customer_contacts (
+			contact_id INT NOT NULL AUTO_INCREMENT,
+            customer_id INT NULL,
```
Comment (adaugochi) [inline submissions/HalimahOladimeji/Database/init.sql:79]: Same here

File: submissions/HalimahOladimeji/Database/init.sql [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+			PRIMARY KEY (item_id),
+			FOREIGN KEY (order_id) REFERENCES orders (order_id),
+			FOREIGN KEY (product_id) REFERENCES products (product_id)
+		);
+	--// Insert records into the order items table
+		INSERT INTO order_items (order_id, product_id, quantity, unit_price, total_amount) 
+			VALUES  (1, 1, 5, 265000.00, 1325000.00),
+				(2, 2, 1, 350000.00, 350000.00),
+				(3, 2, 10, 350000.00, 3500000.00),
+                (3, 1, 2, 265000.00, 530000.00);
```
Comment (adaugochi) [inline submissions/HalimahOladimeji/Database/init.sql:133]: And here

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Comment by HalimahOladimeji** [comment]
Corrected

**Review by adaugochi — APPROVED** [review]

**Pull request merged**